label_sync/labels.yaml

```diff
@@ -1878,6 +1878,12 @@ repos:
         target: both
         prowPlugin: label
         addedBy: anyone
+      - color: e11d21
+        description: PR is missing an area label
+        name: do-not-merge/needs-area
```

Comment:
Same comment as above for `label_sync/labels.md` file, we add 10 labels under `missingLabels`, and here only `do-not-merge/needs-area` is added.

Reply:
Rephrased questions after @killianmuldoon tip here https://github.com/kubernetes/test-infra/pull/30314/files#r1287239715:

Should all the labels that are added to `missingLabels` be also added to `labels.yaml`?

Reply:
The other labels are inherited, earlier in config/prow/config.yaml, starting at line 624 a set of missingLabels was already defined for many repos, including cluster-api. I'm not sure if I needed to copy the other labels down or can just specify the one new label in the section, but have asked in #prow for clarification. Either way though, I would say we do not need to document them here in labels.yaml